Jenkinsfile

```diff
@@ -145,12 +158,9 @@ pipeline {
                                 }
                                 build.untilEach {
                                     /* Wait until a build object is complete */
-                                    echo ">>>>> Enter Build Status Check <<<<<"
```

Comment:
If you're ok with it I'd vote to leave these in... pipeline logs are scarce enough... but up to you :smile: